xsd/netex_framework/netex_reusableComponents/netex_equipmentVehiclePassenger_support.xsd

```diff
@@ -53,6 +60,19 @@ Rail transport, Roads and Road transport
 	</xsd:annotation>
 	<!-- ======================================================================= -->
 	<!-- ======================================================================= -->
+	<xsd:complexType name="vehicleEquipments_RelStructure">
```

Comment:
Already exists and even in an extended form:
	<xsd:complexType name="vehicleEquipments_RelStructure">
		<xsd:annotation>
			<xsd:documentation>List of VEHICLE EQUIPMENT.</xsd:documentation>
		</xsd:annotation>
		<xsd:complexContent>
			<xsd:extension base="containmentAggregationStructure">
				<xsd:choice maxOccurs="unbounded">
					<xsd:element ref="AccessVehicleEquipment"/>
					<xsd:element ref="WheelchairVehicleEquipment"/>
				</xsd:choice>
			</xsd:extension>
		</xsd:complexContent>
	</xsd:complexType>
	<!-- ====ACCESS=====